src/collectors/dropwizard/dropwizard.py

```diff
@@ -32,65 +102,53 @@ def get_default_config(self):
         """
         config = super(DropwizardCollector, self).get_default_config()
         config.update({
-            'host':     '127.0.0.1',
-            'port':     8081,
-            'path':     'dropwizard',
+            'host': '127.0.0.1',
+            'port': 8081
         })
         return config
 
     def collect(self):
         if json is None:
             self.log.error('Unable to import json')
             return {}
-        url = 'http://%s:%i/metrics' % (
-            self.config['host'], int(self.config['port']))
+
+        url = 'http://{}:{}/metrics'.format(self.config['host'],
+                                            self.config['port'])
         try:
             response = urllib2.urlopen(url)
```

Comment:
switch to `requests.get` ..? better than `urllib2`

Reply:
Sure, updated.